reporting/tests/report.py

```diff
@@ -120,88 +119,159 @@ def run_km_job(url, scenario, task_name):
             success = False
             logging.error(f"{task_name} job: {job_id} - status: failed")
             break
-    
 
     execution_time = time() - start_time
-    result.update({
-        "time" : execution_time,
-        "accuracy" : None,
-        "success" : success
-    })
+    result.update({"time": execution_time, "accuracy": None, "success": success})
     return result
 
 
+def non_applicable_run(task_name):
+    return (task_name, {"success": None, "time": 0, "accuracy": None})
```

Comment:
Should `time` also be `None`?

Reply:
@fivegrant we are keeping time as a number for sake of compatibility with integration dashboard? Should we change this to `None` so no runtime shows up?

Reply:
https://github.com/DARPA-ASKEM/integration-dashboard/commit/ea98617bef8c5c618699ea9d3e957fec7cb23671
we can now handle nulls

Reply:
Changed to `None` with changes to integration dashboard also happening to support this.